### PR TITLE
[RF-755] Documents and Sentry

### PR DESCRIPTION
**Pull request by ksalmon**
**Added the Following**
- Document and Agreements
- Sentry Logs
- Github actions for deploys (no longer through netlify cli)

### CODE REVIEW

File: contexts/Documents.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+const initialState = {
+    isLoading: false,
+    documents: [],
+};
+
+export interface DocumentsProviderProps {
+    children?: JSX.Element;
+}
+
+export class DocumentsProvider extends React.Component<DocumentsProviderProps, DocumentProviderState> {
```
Comment (RockinRonE) [inline contexts/Documents.tsx:38]: What's the reason for favoring classes over functional components?

File: screens/StatementScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import Button from '../components/Button';
+import Screen from '../components/Screen';
+import { Heading3, Heading4,  Body } from '../components/Typography';
+import { RootStackParamList } from '../types';
+import { useDocuments } from '../contexts/Documents';
+import { Document } from '../models';
+import TextLink from '../components/TextLink';
+import { isEmpty } from 'lodash';
+import utils from '../utils/utils';
+
```
Comment (RockinRonE) [inline screens/StatementScreen.tsx:14]: Do you want to prepend "I" to the interface names?  😉

**Review by RockinRonE — APPROVED** [review]

**Review by KramerJeff — APPROVED** [review]

File: contexts/Documents.tsx (hunk 1 above)
Comment (ksalmon) [reply contexts/Documents.tsx:38]: It's been the pattern designated from priors. Didn't want to break any patterns this soon into the projects lifecycle :/

Comment (ksalmon) [reply contexts/Documents.tsx:38]: I'm normally a function first approach like in admin

**Pull request merged**